src/main/java/org/threeten/extra/chrono/InternationalFixedDate.java

```diff
@@ -0,0 +1,624 @@
+ * <p>
+ * It provides for a year of 13 months of 28 days each, with one or two days a year belonging to no month or week.
+ * It is therefore a perennial calendar, with every date fixed always on the same weekday.
+ * Though it was never officially adopted in any country, it was the official calendar of the Eastman Kodak Company
+ * from 1928 to 1989.</p>
+ * <p>
+ * This date operates using the {@linkplain InternationalFixedChronology International fixed calendar}.
+ * This calendar system is a proposed reform calendar system, and is not in common use.
+ * The International fixed differs from the Gregorian in terms of month count and length, and the leap year rule.
+ * Dates are aligned such that {@code 0001-01-01 (International fixed)} is {@code 0000-12-31 (ISO)}.</p>
+ * <p>
+ * More information is available in the <a href='https://en.wikipedia.org/wiki/International_Fixed_Calendar'>International fixed Calendar</a> Wikipedia article.</p>
+ * <p>
+ * <h3>Implementation Requirements</h3>
+ * This class is immutable and thread-safe.
+ * <p>
+ * This class must be treated as a value type. Do not synchronize, rely on the
+ * identity hash code or use the distinction between equals() and ==.</p>
+ */
+public final class InternationalFixedDate
+        extends AbstractDate
+        implements ChronoLocalDate, Serializable {
+
+    /**
+     * Serialization version UID.
+     */
+    private static final long serialVersionUID = -7473722012599657263L;
+
+    /**
+     * The days per 400 year cycle.
+     */
+    private static final int DAYS_PER_CYCLE = 146097;
+
+    /**
+     * Number of years in a decade.
+     */
+    private static final int YEARS_IN_DECADE = 10;
+
+    /**
+     * Number of years in a century.
+     */
+    private static final int YEARS_IN_CENTURY = 100;
+
+    /**
+     * Number of years in a millennium.
+     */
+    private static final int YEARS_IN_MILLENNIUM = 1000;
+
+    /**
+     * The proleptic year.
+     */
+    private final int prolepticYear;
+
+    /**
+     * The month.
+     */
+    private final short month;
+
+    /**
+     * The day.
+     */
+    private final short day;
+
+    /**
+     * In a leap year, is it the day between June 28th and Sol 1st ?
+     */
+    private final boolean isLeapDay;
+
+    /**
+     * Is is the last day of the year ?
+     */
+    private final boolean isYearDay;
+
+    //-----------------------------------------------------------------------
+
+    /**
+     * Creates an instance from validated data.
+     *
+     * @param prolepticYear the International fixed proleptic-year
+     * @param month         the International fixed month, from 1 to 13
+     * @param dayOfMonth    the International fixed day-of-month, from 1 to 28, the 29th is only legal for leap day and year day
+     */
+    private InternationalFixedDate (final int prolepticYear, final int month, final int dayOfMonth) {
+        ChronoField.YEAR.checkValidValue (prolepticYear);
```

Comment:
`ChronoField.YEAR` is going to allow negative values.  You created your own `YEAR_RANGE`, right?  I personally am leery of including anything with a decent chance of throwing an exception in a constructor, because you can get into weird places if the caller catches it (the reference might be returned, which means you may end up with a sorta-useable object...); this is part of what the `.create(...)` method was for.